lib/exercises.rb

```diff
@@ -2,19 +2,67 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n) where n is the number of strings for the hashing function but I'm not sure how split/sort/join factors in (maybe O of m where m is the number of letters in all the strings?). 
+# Space Complexity: O(n) because we are creating a new place in memory for each string
 
 def grouped_anagrams(strings)
```

Comment:
If the strings are not arbitrarily short, then this would be O(n * m log m) where n is the number of strings and m is the length of each word.